templates/clicker-game-tg-mini-app-template/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "clicker-game-tg-mini-app-template",
+  "private": true,
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "scripts": {
+    "move:init": "node ./scripts/init",
+    "move:test": "node ./scripts/move/test",
+    "move:compile": "node ./scripts/move/compile",
+    "move:publish": "node ./scripts/move/publish",
+    "move:upgrade": "node ./scripts/move/upgrade",
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "deploy": "vercel",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
+    "_fmt": "prettier 'frontend/**/*.ts'",
+    "fmt": "npm run _fmt -- --write"
+  },
+  "dependencies": {
+    "@aptos-labs/ts-sdk": "1.26.0",
+    "@aptos-labs/wallet-adapter-react": "^3.5.10",
+    "@mizuwallet-sdk/core": "^1.3.0",
```

Comment:
I believe some dependencies are based on the signing method selection - could we try generate the `package.json` file based on that also?
For reference https://github.com/alchemyplatform/create-web3-dapp/blob/main/helpers/utils/generatePackageDotJson.ts